### PR TITLE
:sparkles: Add CI/CD

### PR DESCRIPTION
**Pull request by MarcosOliveir4**

### CODE REVIEW

**Comment by iurynogueira** [comment]
Change node version to v12.22.1

**Review by mizek1 — APPROVED** [review]
Looks good.

File: .gitignore [hunk 1]
```diff
@@ -44,5 +44,5 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
-yarn.lock
+# yarn.lock
```
Comment (iurynogueira) [inline .gitignore:47]:
```suggestion
yarn.lock
```

File: .gitignore [hunk 2]
```diff
@@ -44,5 +44,5 @@ testem.log
...
 package-lock.json
```
Comment (iurynogueira) [inline .gitignore:48]: Remove file

**Review by iurynogueira — CHANGES_REQUESTED** [review]

**Review by iurynogueira — APPROVED** [review]

**Pull request merged**